.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "lib/i18next"]
 	path = lib/i18next
 	url = https://github.com/i18next/i18next.git
+[submodule "lib/bootstrap"]
+	path = lib/bootstrap
+	url = https://github.com/twbs/bootstrap-sass/
+[submodule "lib/magnific-popup"]
+	path = lib/magnific-popup
+	url = https://github.com/dimsemenov/Magnific-Popup
```

Comment:
I agree it's easier to make necessary changes to those projects, but this also comes at a cost - there is a divergence growing between your version of OTR, for example, and the upstream one. If more and more patches are not included in the main distribution (even if most are accepted, let's suppose), the burden of maintaining those changes will grow larger and larger.

Other than these comments above, everything looks fine. Good job!